modules/datastore/src/Service/Import.php

```diff
@@ -21,9 +20,19 @@ class Import {
   use LoggerTrait;
   use EventDispatcherTrait;
 
-  const EVENT_CONFIGURE_PARSER = 'dkan_datastore_import_configure_parser';
+  /**
+   * Event name used when configuring the parser during import.
+   *
+   * @var string
+   */
+  protected const EVENT_CONFIGURE_PARSER = 'dkan_datastore_import_configure_parser';
```

Comment:
```suggestion
  public const EVENT_CONFIGURE_PARSER = 'dkan_datastore_import_configure_parser';
```
Event constants should be public.